containers/packetfence-perl/debian/Dockerfile_debian

```diff
@@ -0,0 +1,61 @@
+FROM debian:11
+
+ARG workdir \
+    output_directory
+
+ENV WORKDIR=${workdir} \
+    OUTPUT_DIRECTORY=${output_directory} \
+    BASE_DIR=/usr/local/pf/lib/perl_modules
+
+
+ENV PERL5LIB=/root/perl5/lib/perl5:${BASE_DIR}/lib/perl5/ \
+    PKG_CONFIG_PATH=/usr/lib/pkgconfig/ \
+    CPAN_BIN_PATH="/usr/bin/cpan" \
+    CPAN_VERSION=2.36
+
+WORKDIR ${WORKDIR}
+
+RUN  apt update -y && \
+    apt install libmodule-signature-perl zip make build-essential \ 
+libssl-dev zlib1g-dev libmariadb-dev-compat libmariadb-dev libssh2-1-dev libexpat1-dev \
+pkg-config libkrb5-dev libsystemd-dev libgd-dev libcpan-distnameinfo-perl libyaml-perl \
+curl wget graphviz libio-socket-ssl-perl debhelper \ 
+libnet-ssleay-perl libcpan-perl-releases-perl python3 vim -y && \
+    apt-get clean && \ 
+    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /tmp/* /var/tmp/*
+
+#RUN TestPerlConfig=$(perl -e exit); [[ ${TestPerlConfig} != "" ]] && export LC_CTYPE=en_US.UTF-8;  export LC_ALL=en_US.UTF-8
```

Comment:
If not used then remove the line or add a comment before it to explain why it's here.